.circleci/config.yml

```diff
@@ -642,6 +642,198 @@ jobs:
     steps:
       - checkout
 
+  importers:
```

Comment:
i think you probably can follow the pattern we have for unit-tests/system-tests and split the executor from the job and (maybe? you could even re-use the system job)

Reply:
How about something like this?
https://github.com/wandb/wandb/pull/6897/files/624ddb1da0c15bd8c7fc646a4d09c4d97c50f192..bedf47df9bed2d65077a3b00a82e7d2eb10decde